minecraft_servers/surnex/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "server_name": "surnex",
+  "nice_name": "SurnexNET",
+  "direct_ip": "surnex.net",
+  "social": {
+    "web": "https://surnex.net",
+    "twitter": "SurnexNET",
+    "discord": "https://dc.Surnex.net",
```

Comment:
As custom discord invite links are only supported for LabyMod partners at the moment.
Please use a discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url